From e11147b43cc4fe029af6e71df4de2fce0db28383 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 7 Sep 2018 01:27:49 +0530
Subject: [PATCH 635/909] arm64: dts: r8a77990-ebisu: Add vspb and vspi0 for
 VSP Manager

commit 66b4b098b764ecd63906cad83889aab3864349f7 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
index fd5743148132..508c35642953 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
@@ -315,6 +315,10 @@
 		enable-active-high;
 	};
 
+	vspm_if {
+		compatible = "renesas,vspm_if";
+	};
+
 	x12_clk: x12 {
 		compatible = "fixed-clock";
 		#clock-cells = <0>;
@@ -806,6 +810,14 @@
 	status = "okay";
 };
 
+&vspb {
+	status = "okay";
+};
+
+&vspi0 {
+	status = "okay";
+};
+
 &xhci0 {
 	pinctrl-0 = <&usb30_pins>;
 	pinctrl-names = "default";
-- 
2.17.1

